src/models/backup.js

```diff
@@ -326,6 +338,12 @@ export default {
     changeTagsLoading(state, action) {
       return { ...state, ...action.payload }
     },
+    ensureRouteLocation(state, action) {
```

Comment:
Do we need this?

Reply:
This method should indeed be deleted. This is the code left over when I try to modify it in another way. Thanks.